lib/presentation/views/home/home.dart

```diff
@@ -2,17 +2,22 @@ import 'package:flutter/material.dart';
 import 'package:sis_group6/mock/sentiment_data.dart';
 import 'package:sis_group6/presentation/widgets/sentiment_chart.dart';
 
+import '../../widgets/word_cloud.dart';
```

Comment:
Prefer importing with the full path: import `'package:sis_group6/presentation/widgets/word_cloud.dart';`

Reply:
Addressed in previous commits